### DISCUSSION TITLE
Convert Microsoft phi 1.5

### DISCUSSION THREAD

**Discussion by loretoparisi** [ggml-org/llama.cpp#3286] [Q&A] [answered]
I'm trying to convert to GGUF the new [phi-1_5](https://huggingface.co/microsoft/phi-1_5/tree/main)

```shell
python3 convert.py /Downloads/phi-1_5 
Loading model file /Downloads/phi-1_5/pytorch_model.bin
Traceback (most recent call last):
  File "/llama.cpp/convert.py", line 1196, in <module>
    main()
  File "/llama.cpp/convert.py", line 1145, in main
    params = Params.load(model_plus)
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/llama.cpp/convert.py", line 299, in load
    params = Params.loadHFTransformerJson(model_plus.model, hf_config_path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/llama.cpp/convert.py", line 209, in loadHFTransformerJson
    n_embd           = config["hidden_size"]
                       ~~~~~~^^^^^^^^^^^^^^^
KeyError: 'hidden_size'
```

Model Card is

- Architecture: Transformer-based model with next-word prediction objective
- Dataset size: 30B tokens
- Training tokens: 150B tokens
- Precision: fp16
- GPUs: 32xA100-40G
- Training time: 8 days

**Comment by ianscrivener** [comment, marked as answer]
Phi 1.5 is not (yet) a supported model [README.md](https://github.com/ggerganov/llama.cpp).

There is already an request in to add Phi 1.5 support [[3139]](https://github.com/ggerganov/llama.cpp/discussions/3139).
